### ISSUE TITLE
I got probelm when I run TE Aid in multiple threads 

### ISSUE THREAD

**Issue by qjiangzhao** [clemgoub/TE-Aid#5]
Hi,

Many thanks for the wonderful tool. 

I integrated TE Aid into my software. My tool enables multiple threads, but I got a problem when I run TE Aid in multiple threads. 

I found TE Aid will create blast database every time. Is that right?

Here you can find the error

Error encountered: /work/DNA_DTCfaBLAST Database error: No alias or index file found for nucleotide database [/work/rwth0146/Rice/Rice_genome/all.chrs.con] in search path [/rwthfs/rz/cluster/work/ur376715/TE_analysis/Software_develpment_for_TE_curation/version_24/bin::]
Error in read.table(text = system(paste("blastn -query", query, "-db",  : 
  no lines available in input
Calls: consensus2genome -> read.table
Execution halted

Yours sincerely
Jiangzhao

**Comment by qjiangzhao** [comment]
I found from your code that the genome database will be created every time. It will be better to judge if the database exists in your code. Then it will use less computational resources and all the multiple threads execution.

**Comment by clemgoub** [comment]
Dear Jiangzhao,

Thanks for your kind message and feedback. Indeed, this is pretty wasteful to create a genome database each time. I will look into it as soon as I can (but I am swamped this week). Meanwhile, if you'd like to implement this feature (should be easy around lines 216-217 of the `TE-Aid` script), feel free to send a pull request and I can merge your changes.

Best,

Clément

**Comment by clemgoub** [comment]
Regarding the error you get in your pipeline, do you think it is caused by this, or something else? I would need more context to understand what is going on. Maybe the command you are running, it looks like a Nextflow pipeline?

Cheers,

Clément

**Comment by qjiangzhao** [comment]
Hi,

If I am right, the genome database will be created every time by TE Aid. It will be better to judge if the database exists in your code. Then it will use less computational resources and allow the multiple threads execution.

**Comment by qjiangzhao** [comment]
Hi Clément,

Many thanks for your quick response.
I am writing a software to replace the TE manual curation, I used TE-Aid to plot my final consensus sequence. 

I modified your code to suppress TE Aid to generate the genome blast database every time. Meanwhile, I generate blast database in my code. After that, the error disappeared. 

Sincerely
Jiangzhao

![image](https://github.com/clemgoub/TE-Aid/assets/75024559/9b206946-5e91-4e26-bff2-9e2bd730be40)
